Tools/MiniBrowser/mac/Info.plist

```diff
@@ -56,6 +56,13 @@
 	<string>Recording video on behalf of websites</string>
 	<key>NSMicrophoneUsageDescription</key>
 	<string>Recording audio on behalf of websites</string>
+	<key>LSEnvironment</key>
+	<dict>
+		<key>__XPC_DYLD_FRAMEWORK_PATH</key>
+		<string>${BUILT_PRODUCTS_DIR}</string>
+		<key>DYLD_FRAMEWORK_PATH</key>
+		<string>${BUILT_PRODUCTS_DIR}</string>
```

Comment:
Some comments:
- Baking a full path into the product means that it's non-relocatable. Notably, MiniBrowser won't work in archives downloaded from https://webkit.org/build-archives/. Can we use a relative path?
- I think that we normally bake the paths into Mach-O load command in the binary, not in the plist.

Reply:
> Baking a full path into the product means that it's non-relocatable. Notably, MiniBrowser won't work in archives downloaded from https://webkit.org/build-archives/. Can we use a relative path?

I had Brady verify that this approach did not affect setting DYLD_FRAMEWORK_PATH and invoking the binary directly. So I expect `run-minibrowser`, etc. using archives to continue to work.